### PR TITLE
add support for tanglegram

### PR DESCRIPTION
**Pull request by taylorreiter**

### CODE REVIEW

File: KEGGDecoder/KEGG_decoder.py [hunk 1]
```diff
@@ -1229,6 +1282,7 @@ def main():
 	parser.add_argument('-o', '--output', help="List version of the final heat\
 						map figure")
 	parser.add_argument('-v', '--vizoption', help="Options: static, interactive, tanglegram")
+        parser.add_argument('--newick', help="Required input for tanglegram visualization")
```
Comment (luizirber) [inline KEGGDecoder/KEGG_decoder.py]:
```suggestion
	parser.add_argument('--newick', help="Required input for tanglegram visualization")
```

File: KEGGDecoder/KEGG_decoder.py [hunk 2]
```diff
@@ -1377,6 +1431,8 @@ def main():
 	if arg_dict['vizoption'] == 'interactive':
 		from .Plotly_viz import plotly_viz
 		plotly_viz(genome)
+        if arg_dict['vizoption'] == 'tangelgram':
```
Comment (luizirber) [inline KEGGDecoder/KEGG_decoder.py]:
```suggestion
        if arg_dict['vizoption'] == 'tanglegram':
```

**Comment by taylorreiter** [comment]
@luizirber is my strategy here ok? The tab vs. spaces was a bit of a nightmare...I reformatted all of the tabs to spaces in the whole document. 

@bjtully...that also means I switched the tabs to spaces, which you may not feel comfortable with in the long run. It looks quite a bit different, but I think its a more standard formatting. However, we can come up with another strategy to put this into the code if you'd like!

**Comment by bjtully** [comment]
@taylorreiter I have very strong opinions about the changes from tabs to spaces...

**Comment by luizirber** [comment]
I'll have to disagree with both =]

I prefer spaces to tab and think choosing tabs make it harder to have contributions, but if these are the conventions adopted in this repo then we need to adapt to it.

@taylorreiter let's say you receive a PR that adds a feature, but changes a thousand lines of (unrelated) code along the way. How would you review it? How long does it take to figure out if a line changed because of formatting, or if it is something that changes the behavior of the code?

**Comment by taylorreiter** [comment]
I'll switch it back! Now that I know I can get all checks to pass, I'll write the same thing with tabs only and submit a new PR

**Pull request merged**